JetNews/app/src/main/res/drawable/icon_article_background.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Copyright header.

Reply:
Done

Reply:
This file is missing `<?xml version="1.0" encoding="utf-8"?>`

Reply:
Added it